testingfacade/DataStore.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (C) 2021 Advanced Media Workflow Association
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import json
+
+
+class DataStore:
+    """
+    Store json with test question details for use with Controller test suite and Testing Facade
+    """
+
+    def __init__(self):
+        self.test_type = None
+        self.question_id = None
+        self.name = None
+        self.description = None
+        self.question = None
+        self.answers = None
+        self.time_sent = None
+        self.timeout = None
+        self.url_for_response = None
+        self.answer_response = None
+        self.time_answered = None
+        self.status = "Empty"
+        self.metadata = None
+
+    def clear(self):
+        self.test_type = None
+        self.question_id = None
+        self.name = None
+        self.description = None
+        self.question = None
+        self.answers = None
+        self.time_sent = None
+        self.timeout = None
+        self.url_for_response = None
+        self.answer_response = None
+        self.time_answered = None
+        self.status = "Empty"
+        self.metadata = None
+
+    def getStatus(self):
+        return self.status
+
+    def setJson(self, json_str):
```

Comment:
Why `_str`? If I've understood correctly, it's not a string-containing-serialized-JSON nor a JSON string, it's an object. Just 'json', maybe?